src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -496,6 +496,7 @@ private async Task ProcessIncomingFramesAsync()
                 catch (HttpProtocolException e)
                 {
                     InitialSettingsReceived.TrySetException(e);
+                    LogExceptions(InitialSettingsReceived.Task);
```

Comment:
InitialSettingsReceived isn't guaranteed to be await'd/consumed elsewhere?

Reply:
No, we'll only look at it if/when we're sending an H2 WebSocket request.